File: Exemplos M10/App_IntentDigitalHUB_ReactNative/Elgin_IntentDigitalHUB_ReactNative-font/src/intentDigitalHubService/BRIDGE/SetSenha.ts
import {BridgeCommand} from './BridgeCommand';

export class SetSenha extends BridgeCommand {
  readonly senha: string;
  readonly habilitada: boolean;

  constructor(senha: string, habilitada: boolean) {
    super('SetSenha');
    this.senha = senha;
    this.habilitada = habilitada;
  }

  functionParameters(): object {
    return {senha: this.senha, habilitada: this.habilitada};
  }
}

File: Exemplos M10/App_IntentDigitalHUB_XamarinForms/Elgin_IntentDigitalHUB_XamarinForms-font/E1_Digital_Hub_Xamarin_Forms_M8/IntentServices/Bridge/SetSenha.cs
namespace Xamarin_Forms_Intent_Digital_Hub.IntentServices.Bridge
{
    class SetSenha : BridgeCommand
    {
        private readonly string senha;
        private readonly bool habilitada;

        public SetSenha(string senha, bool habilitada) : base("SetSenha")
        {
            this.senha = senha;
            this.habilitada = habilitada;
        }

        protected override string FunctionParameters()
        {
            return "\"senha\"" + ":" + "\"" + senha + "\"" + "," +
                    "\"habilitada\"" + ":" + habilitada;
        }
    }
}